pages/board/index.tsx

```diff
@@ -1,29 +1,42 @@
 import { useQuery } from '@apollo/client';
-import { Row } from 'react-bootstrap';
+import { faPlus } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Button, Row, Spinner } from 'react-bootstrap';
+import { DragDropContext } from 'react-beautiful-dnd';
 import BoardSection from '../../components/boardSection';
 import Task from '../../components/task';
 import tasksQuery from './tasks';
 
 const Board = () => {
-  const { data, loading } = useQuery(tasksQuery, {
+  const { data } = useQuery(tasksQuery, {
     onCompleted: (data) => console.log(data),
   });
 
   const sections: Array<string> = ['Backlog', 'In-progress', 'Review', 'Done'];
 
+  const onDragEnd = (d) => {
+    const { destination, source, draggableId } = d;
```

Comment:
Why do we need this variables?

Reply:
These variables will be used later when I move a task component from one boardSection(source) to another boardSection(destination)